product-listing/product-listing.css

```diff
@@ -0,0 +1,223 @@
+@import url(/main.css);
+
+* {
+
+
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+
+}
+
+
+.main-wrapper {
+
+    display: grid;
+    height: 100vh;
+    grid-gap: 2rem;
+    grid-template-columns: 1fr 7fr;
+    padding-bottom: 0.5rem;
+    grid-template-areas: "header header" "aside main";
+    margin-top: 1rem;
+    padding-right: 1.5rem;
+
+}
+
+.flex-column {
+
+    display: flex;
+    flex-direction: column;
+}
+
+.filter-head {
+
+    display: flex;
+    justify-content: space-between;
+}
+
+.filter-spaces {
+
+    margin: 0.5rem 0;
+    padding: 0;
+}
+
+.filter-headings {
+
+    padding: 0.5rem 0;
+}
+
+.filter-clear {
+
+    color: var(--primary-color);
+
+    text-decoration: none;
+    font-weight: 600;
+    font-size: 1.2rem;
+    align-self: center;
+}
+
+.filter-price-range {
+    display: flex;
+    justify-content: space-between;
+}
+
+input {
+
+    font: inherit;
+}
+
+.filter-category .inp-checkbox {
+
+    display: flex;
+    /* justify-content: flex-start; 
+    align-items: center; */
+    column-gap: 0.6rem;
+}
+
+.input-checkbox {
+
+    width: 20px;
+    height: 20px;
+    margin-top: 1rem;
+}
+
+.checkbox-notify {
+
+    font-size: 0.9rem;
+    padding-top: 1.1rem;
+}
+
+.rating-options label {
+    font-size: 0.9rem;
+}
+
+.item-conatiner {
+
+    grid-area: main;
+}
+
+.item-heading {
+
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    column-gap: 0.5rem;
+}
+
+.item-heading p {
+
+    font-weight: bold;
+}
+
+.item-list {
+
+    display: grid;
+    height: 100vh;
+    grid-gap: 1.5rem;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    margin-top: 0.5rem;
+}
+
+.card-image {
+
+    display: flex;
+}
+
+.card-vertical {
+
+    position: relative;
+    box-shadow: 0px 0px 5px 5px lightgray;
+    border-radius: 5px;
+}
+
+.card-vertical-info {
+
+    position: relative;
+    justify-content: flex-start;
+    margin: 0.5rem;
+    flex-wrap: wrap;
+}
+
+.flex-column {
+
+    display: flex;
+    flex-direction: column;
+}
+
+.card-image .adjust-image {
+    height: 48vh;
+    width: 100%;
+    object-fit: contain;
+}
+
+
+.card-text {
+
+    font-size: 0.9rem;
+    font-weight: 600;
+    margin-top: 0.9rem;
+    margin-left: 0.5rem;
```

Comment:
Improvement: It can be done like margin: 0.9rem 0 0 0.5rem

Reply:
Thanks, Gopal. I've now used the shorthand property.